styles.css

```diff
@@ -84,10 +83,430 @@
+  padding: 0;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.see-project:hover {
+  color: blue;
+}
+
+.diffrent-work {
+  display: grid;
+  grid-template-rows: 6fr 6fr 6fr 6fr 6fr 6fr;
+  justify-content: space-evenly;
+  row-gap: 50px;
+  padding-top: 20px;
+  padding-left: 20px;
+  padding-right: 20px;
+  margin-top: 40px;
+  margin-bottom: 0;
+}
+
+.worktwo {
+  display: flex;
+  flex-direction: column;
+  height: 350px;
+  border-radius: 0;
+  background: url("/img/Img\ Placeholder1.svg");
+  gap: 10px;
+  background-repeat: no-repeat;
+  background-size: cover;
+  margin-bottom: 0;
+  margin-top: 50px;
+}
+
+.work-two-title {
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  padding-left: 20px;
+  padding-top: 20px;
+  color: #fff;
+  flex: none;
+  order: 0;
+  margin: 30px 20px 0 15px;
+}
+
+.work-two-dis {
+  font-family: 'Inter', sans-serif;
+  margin-left: 15px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  padding-left: 20px;
+  color: #fff;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin-right: 20px;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.languages-work {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 12px;
+  width: 236px;
+  height: 40px;
+  margin-bottom: 0;
+  margin-right: 70px;
+  margin-top: 0;
+  list-style-type: none;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+}
+
+.code-name {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  margin: 0;
+  color: #fff;
+}
+
+.html-box {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px;
+  height: 20px;
+  background: rgba(255, 255, 255, 0.24);
+}
+
+.bootstrap-box {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  height: 20px;
+  background: rgba(255, 255, 255, 0.24);
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.ruby-box {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  height: 20px;
+  background: rgba(255, 255, 255, 0.24);
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+}
+
+.see-two {
+  text-decoration: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 19px;
+  line-height: 24px;
+  color: #fff;
+  border: none;
+  order: 3;
+  background-color: #ff6b00;
+}
+
+.see-two:hover {
+  color: blue;
+}
+
+.see-two-box {
+  flex-direction: row-reverse;
+  display: flex;
+  justify-content: center;
+  align-items: stretch;
+  margin-bottom: 0;
+  margin-right: 0;
+  order: 3;
+  background-color: #ff6b00;
+  width: 350px;
+  height: 90px;
+  background-size: cover;
+}
+
+.aboutmyself {
+  background: #1c1a19;
+}
+
+.aboutme {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 24px;
+  gap: 24px;
+  background-image: url("/img/Group.svg");
```

Comment:
[REQUIRED] Kindly fix the link as suggested by the previous reviewer

Reply:
i don't know the issue is where the first code reviewer didn't have a problem I received this from git does it play a role in the issue
![Screenshot (7)](https://user-images.githubusercontent.com/103601335/192115281-376514bf-08d8-483c-b940-938d0519fc3f.png)